rust-connectors/sinks/kafka/src/main.rs

```diff
@@ -165,30 +174,24 @@ impl KafkaSinkDependencies {
                 }
                 (_, Some(ca_pem)) => {
                     let mut tmpfile = NamedTempFile::new().unwrap();
+
                     write!(tmpfile, "{}", ca_pem).unwrap();
-                    let path = tmpfile.into_temp_path();
-                    path.persist("/tmp/kafka-client-ca.pem")?;
-                    client_config.set("ssl.ca.location", "/tmp/kafka-client-ca.pem");
+                    let (_file, path) = tmpfile.keep()?;
```

Comment:
In which release of rdkafka this will not be needed?

Reply:
I believe >= to `0.27.0`.